docroot/themes/humsci/su_humsci_theme/patterns/molecules/featured-card/scss/featured-card.scss

```diff
@@ -0,0 +1,35 @@
+@charset "UTF-8";
+
+//
+// Featured Card
+//
+
+// Decanter
+@import
+  'decanter-no-markup';
+
+.featured-card__container {
+  .featured-card__img {
+    position: relative;
+
+    img {
+      width: 100%;
```

Comment:
I don't like this. it could cause small images to stretch.. without this, it would allow the user to choose which image size to provide using image styles.

Reply:
Still thinking about this. The height will grow automatically (we will not hard code that) depending on the width. I don't like the potential pixelation, either. Maybe we allow for larger image sizes only that then scale when used with a decanter grid class, so that the pixelation is never bad. I don't want to hardcode a width on the content container either, in case the image style changes. This felt like the most global option, as I also don't like how the card looks with an image that only spans its natural width when the content container stretches 100%.

Reply:
![screen shot 2018-06-19 at 1 34 07 pm](https://user-images.githubusercontent.com/7185045/41622790-8a942002-73c5-11e8-8141-bf845dbfef0b.png)
I removed the image width 100% and applied `width: fit-content;` to the wrapper with the box shadow.

Reply:
Nice! The image needs to be edge-to-edge, though. Wondering if we can use your solution, but remove the padding from the image.

Reply:
No luck so far. Also, not only do we lose the edge-to-edge image, but the fit content for a container will find the largest but of content. See what happens when I update the title of the card:
![screen shot 2018-06-19 at 2 10 34 pm](https://user-images.githubusercontent.com/8933670/41624444-90251d5a-73ca-11e8-9ac0-283cadcac66f.png)

Reply:
well thats better IMO. we will have something in place to make them have 2 or 3 side by side. but at least this way the image doesnt stretch. a more ideal thing for the users i think. but its your front end too sooo.... dont let me dictate it :)